src/css/custom.css

```diff
@@ -194,23 +194,450 @@ table {
+  justify-content: start;
+  flex-wrap: wrap;
+  margin-top: 45px;
+  gap: 10px 15px;
+}
+
+@media (max-width: 427px) {
+  .hero__CTA-buttons-container {
+    margin-top: 30px;
+    flex-direction: column;
+    justify-content: center;
+  }
+}
+
+.featured-case__section {
+  padding: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-top: 0;
+  min-height: calc(100vh - 60px);
+  background-size: cover;
+  background-position: center center;
+}
+
+.featured-case__container {
+  width: 100%;
+  padding: 0 16px;
+  overflow: hidden;
+  position: relative;
+}
+
+.featured-case__content-wrapper {
+  padding-top: 60px;
+}
+
+.featured-case__grid-container {
+  width: 100%;
+  padding-bottom: 16px;
+  align-items: start;
+}
+
+@media (min-width: 768px) {
+  .featured-case__content-wrapper {
+    padding: 0 24px 0 0;
+  }
+
+  .featured-case__grid-container {
+    min-width: 700px;
+    padding-top: 90px;
+    display: flex;
+    align-items: center;
+  }
+}
+
+@media (min-width: 1024px) {
+  .featured-case__grid-container {
+    min-width: 1000px;
+  }
+}
+
+.featured-case__image-container__absolute {
+  position: absolute;
+  width: 50%;
+  height: 50%;
+}
+
+.footer-styled {
+  overflow: hidden;
+  padding: 20px 0;
+  background-size: cover;
+  background-position: center center;
+  color: var(--theme-color-stormy-blue);
+}
+
+.footer-styled__top-section {
+  margin-top: 42px;
+  padding-bottom: 56px;
+}
+
+.footer-styled__column-container {
+  padding: 12px;
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-styled__column-container__centered {
+  padding: 12px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-styled__copyright-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 40px;
+}
+
+.footer-styled__showcase-card-container {
+  padding: 1.5em;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-bottom: 1em;
+  border: 1px solid var(--theme-color-stormy-blue);
+}
+
+@media (min-width: 768px) {
+  .footer-styled {
+    padding: 30px 0;
+  }
+
+  .footer-styled__column-container {
+    width: 20%;
+  }
+
+  .footer-styled__showcase-card-container {
+    max-width: 20%;
+  }
+}
+
+.footer-styled__showcase-card-text {
+  text-align: center;
+  padding-top: 1.5em;
+  font-size: 16px;
+}
+
+.footer-styled__column-item {
+  margin-bottom: 10;
+}
+
+.footer-styled__common-text {
+  background-color: inherit;
+  border: none;
+  font-size: 16px;
+  font-weight: 300;
+  color: var(--theme-color-stormy-blue);
+  transition: all 0.2s ease-out;
+}
+
+.footer-styled__common-text:hover {
+  color: var(--theme-color-stormy-blue);
+  cursor: pointer;
+}
+
+.footer-styled__column-button {
+  padding: 0;
+}
+
+.footer-styled__column-link {
+  text-decoration: none !important;
+}
+
+.footer-styled__column-mail {
+  text-decoration: none !important;
+}
```

Comment:
Not needed at all. Without it, it fallbacks to `main a`

Reply:
Same as in https://github.com/vazco/forminer/pull/62#discussion_r1288187776